app/Resources/assets/js/admin.js

```diff
@@ -0,0 +1,69 @@
+var $ = require('jquery');
+window.$ = $;
+window.jQuery = $;
+
+require('bootstrap-sass');
+require('moment');
+require('eonasdan-bootstrap-datetimepicker');
+
+require('imports-loader?define=>false!typeahead.js/dist/typeahead.jquery.min.js');
+const Bloodhound = require('imports-loader?define=>false!typeahead.js/dist/bloodhound.js');
+window.Bloodhound = Bloodhound;
```

Comment:
Again, this shouldn't be needed (in theory), but it's possible there's some legacy library that isn't requiring this correctly. We'll need to make sure each line is needed, and document which lines *are* needed and why